src/IO/ADIOS/ADIOS2IOHandler.cpp

```diff
@@ -1270,7 +1276,17 @@ void ADIOS2IOHandlerImpl::getBufferView(
         return;
     case UseSpan::Auto:
         if (switchAdios2VariableType<detail::HasOperators>(
-                parameters.dtype, name, ba.m_IO))
+                parameters.dtype, name, ba.m_IO)
+#if (                                                                          \
+    ADIOS2_VERSION_MAJOR * 1000 + ADIOS2_VERSION_MINOR * 10 +                  \
+    ADIOS2_VERSION_PATCH) <= 2102
```

Comment:
@franzpoeschel do I read correctly that the span-based API is broken for ADIOS2 of all versions unless ADIOS2 in a version newer than 2.10.2 is used? In other words, we cannot use it just yet in production (unless we use ADIOS2 development) and we do not need any further changes to this check here (with https://github.com/ornladios/ADIOS2/pull/4587 merged)?

Reply:
Ah, see your comment above. Exactly